### DISCUSSION TITLE
[WP4255 Discussion]: missing reference to KEGG on new site

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#73] [pathway-discussion]
### Discussion

On this pathway (https://www.wikipathways.org/pathways/WP4255), there is an empty reference. On the classic page, this references ''KEGG Non-small cell lung cancer''; https://www.kegg.jp/dbget-bin/www_bget?pathway+map05223, 

### Type of revision

- [ ] Major
- [X] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
This is because URLs are currently not supported (see https://github.com/wikipathways/wikipathways.github.io/issues/89). The same issue exists in other pathways, for example https://www.wikipathways.org/pathways/WP4666.html.

**Comment by khanspers** [comment]
I added the URL directly to the descriptions of WP4666 and WP4255 for now.

**Comment by egonw** [comment]
A recent update to how references are handled makes it now look like this:

![image](https://github.com/wikipathways/wikipathways-help/assets/26721/7b8a1983-f6ca-4e7c-85fe-7c6db62fc5e4)

Is uses KEGG as database, and alternatively, `URL` is used as datasource and the URL as identifier. Then the URL will now be visible in the references list. cc @larsgw

**Reply by danidi** [reply]
Thank you! But in this case (https://www.wikipathways.org/pathways/WP4255) it does not give the whole url, but only the ending. So the link is not clickable. Could it be because there is a + symbol in the link?

**Reply by larsgw** [reply]
We have changed it so that the link is clickable now.

**Reply by danidi** [reply]
Great, thank you!
